client/src/components/SharedComponents/Signup/signup.css

```
@@ -92,6 +92,6 @@ button:hover {
   color: red;
 }
 
-.TrueValidate {
+.True-vsalidate {
```

you have a typo here in ```validate```
```suggestion
.true-validate {
```

---

@FarahZaqout  
Done!